projector-awt/src/main/kotlin/org/jetbrains/projector/awt/PToolkit.kt

```diff
@@ -321,6 +321,10 @@ class PToolkit : SunToolkit(), KeyboardFocusManagerPeerProvider, ComponentFactor
     return true
   }
 
+  override fun getNumberOfButtons(): Int {
+    return 5 // this is cached in MouseEvent, so support at least mice with two side buttons
```

Comment:
I suppose it's needed in Swing client, so please specify this explicitly

Reply:
Yep, this could be useful for the web client too, at least we have such a ticket: (PRJ-252 Only mouse buttons 1-3 can be used in keymaps)[https://youtrack.jetbrains.com/issue/PRJ-252]. Let's support it separately after merging the PR.